.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # Raising a PR against the agent spec folder will automatically request reviews from all agent teams
 # If the PR is not ready for review, create a draft PR instead
 # See also /.github/pull_request_template.md
-/specs/agents/mobile @elastic/apm-agent-ios @elastic/apm-agent-android
 /specs/agents @elastic/apm-agent-go @elastic/apm-agent-java @elastic/apm-agent-net @elastic/apm-agent-node-js @elastic/apm-agent-php @elastic/apm-agent-python @elastic/apm-agent-ruby @elastic/apm-agent-rum @elastic/apm-pm
```

Comment:
@elastic/apm-agent-ios @elastic/apm-agent-android do you want to be pinged on the broader agent specs on this line? Since you're based on opentelemetry, I don't know how relevant they are to your work.

Reply:
I think it's still valuable

Reply:
Cool, I added it.